PROJECT

```diff
@@ -0,0 +1,11 @@
+domain: tigera.io
+layout: go.kubebuilder.io/v2
+projectName: operator
```

Comment:
just curious, what's this file about?

Reply:
It was auto created but from some recent reading it is consumed by operator-sdk I believe. One of the things it is used for is the layout of the repo structure.